README.md

```diff
@@ -809,7 +809,17 @@ The GitHub issues are intended for bug reports and feature requests. Keeping the
 
 ## Releases
 
-The AWS JDBC Driver for MySQL has a regular monthly release cadence. A new release will occur during the last week of each month. However, if there are no changes since the latest release, then a release will not occur. 
+The AWS JDBC Driver for MySQL has a regular monthly release cadence. A new release will occur during the last week of each month. However, if there are no changes since the latest release, then a release will not occur.
+
+## Aurora Engine Version Testing
+The `aws-mysql-jdbc` driver is being tested against the following Community and Aurora database versions in our test suite:
+
+| Database        | Versions                                                                                                                                                                                                  |
+| --------------- |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| MySQL           | 8.0.21                                                                                                                                                                                                    |
```

Comment:
We should update this

Reply:
Good point. I'll create a chore ticket to address that.